### PR TITLE
treewide: fix some apk version issues 

### PR DESCRIPTION
**Pull request by feckert**
Maintainer: @robimarko and @aparcar 
Compile tested: x86_64
Run tested: not needed

Description:
Fix apk version issue for:
* libmspack
* bluez-tools
* bonding

### CODE REVIEW

**Comment by aparcar** [comment]
I wonder what the ci doesn't like. Will check

**Comment by feckert** [comment]
@aparcar I've asked myself the same thing
I suspect that the [regex](https://github.com/openwrt/packages/blob/master/.github/workflows/check-apk-valid-version.yml#L50) is wrong.
Shouldn't it be like that?
`^[1-9]\d*(\.[1-9]\d*)*$`

**Comment by aparcar** [comment]
Release must be an integer, version is evaluated by APK

**Comment by aparcar** [comment]
Issue is that the patch is combined to `net/bonding/utils/bluez-tools/libs/libmspack/Makefile` which isn't valid :) I'll have to fix that

**Comment by aparcar** [comment]
Please have a look at https://github.com/openwrt/packages/pull/24807.

**Comment by feckert** [comment]
@aparcar I have rebased my changes on the latest master so I have your CI/CD changes. After that I did a git push --force and the CI/CD still complains!
Did I made something wrong?

File: utils/bluez-tools/Makefile [hunk 1]
```diff
@@ -7,8 +7,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bluez-tools
-PKG_VERSION:=20201025.f653217
-PKG_RELEASE:=2
+PKG_VERSION:=2020.10.25
```
Comment (neheb) [inline utils/bluez-tools/Makefile:10]: Remove PKG_VERSION. It will be autogenerated from PKG_SOURCE_DATE

**Comment by aparcar** [comment]
Could you please rebase it again? It doesn't seem to run the latest version of the CI script.

**Comment by aparcar** [comment]
Well, APK is now happy :)

**Pull request merged**